src/tests/ftest/pool/attribute.py

```diff
@@ -144,11 +149,10 @@ def test_pool_attributes(self):
         """
         self.add_pool()
         expected_for_param = []
-        name = self.params.get("name", '/run/attrtests/name_handles/*/')
+        name = self.params.get("name", '/run/attrtests/test_handles/*/')
         expected_for_param.append(name[1])
-        value = self.params.get("value", '/run/attrtests/value_handles/*/')
+        value = self.params.get("value", '/run/attrtests/test_handles/*/')
```

Comment:
This should not be changed - see comment in yaml file.

Reply:
Fixed